app/helpers/session_helper.rb

```diff
@@ -0,0 +1,40 @@
+def user_at_email(email)
+	User.find_by_email(email)
+end
+
+def user_is_authenticated?(user, password)
+	return false if user.blank?
+	user.authenticate(password) ? true : false
```

Comment:
authenticate returns a `Boolean` value already. You do not need to add `? true : false`